api-metastore/src/main/java/org/zalando/nakadi/controller/EventTypeController.java

```diff
@@ -68,10 +70,10 @@ public EventTypeController(final EventTypeService eventTypeService,
     }
 
     @RequestMapping(method = RequestMethod.GET)
-    public ResponseEntity<?> list() {
-        final List<EventType> eventTypes = eventTypeService.list();
+    public ResponseEntity<?> list(@RequestParam final Optional<String[]> writers) {
```

Comment:
Why not use `Set<>` like we do for `readers` (and `event_type`) in `SubscriptionController`?